doc/modules/ensemble.rst

```diff
@@ -896,6 +895,19 @@ based on permutation of the features.
 
  * :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_regression.py`
 
+.. topic:: References
+
+  .. [F1999] Friedman, J.H. (2002). `Stochastic gradient boosting.
+     <https://statweb.stanford.edu/~jhf/ftp/stobst.pdf>`_.
+     Computational Statistics & Data Analysis, 38, 367-378.
+
+  .. [F2001] Friedman, J.H. (2001). :doi:`Greedy function approximation: A gradient
```

Comment:
From the [doc build error](https://app.circleci.com/pipelines/github/scikit-learn/scikit-learn/25466/workflows/3ec7f283-416e-453b-b7a3-55282570cc7c/jobs/185934) the links are globally scoped. We can be more specific here:

```suggestion
  .. [Friedman2001] Friedman, J.H. (2001). :doi:`Greedy function approximation: A gradient
```